trieste/models/gpflow/optimizer.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2021 The Trieste Contributors
```

Comment:
this is good but gpflow changes should be part of a separate PR

a note: `_create_scipy_optimizer` should probably be moved here as well

Reply:
See #343